Azərbaycan dili tərcümənizi yeniləyin

```diff
@@ -2,7 +2,7 @@
 
     <string name="Players">"Oyunçular"</string>
     <string name="Control_">"Nəzarət:"</string>
-    <string name="CLOSE">"Bağla"</string>
+    <string name="CLOSE">"Bağlayın"</string>
```

Comment:
According to the few grammar rules I know about the Turkic languages, I think turning it as "Bağlayın" will mean "You close". I might be wrong but that's all I know.